docs/configure

```diff
@@ -672,20 +676,20 @@ virtual_domains:
                               { and {{<{$spam_score_int}{${extract{sa_tag}{$address_data}}} } \
                                      {eq {1}{${extract{on_spamassassin}{$address_data}}}} \
                                     } \
-                              } { eq {0}{${extract{on_spamassassin}{$address_data}}}} \
+                              } { eq {0}{${extract{dom_spamassassin}{$address_data}}}} \
```

Comment:
The thing is, you removed `|| on_spamassassin==0` from the condition.
But I think the shorter these conditions, the easier they are to read, so I'm all for the "nice expression" path. :)